packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.scss

```diff
@@ -52,4 +52,19 @@
       border: 2px solid black;
     }
   }
+
+  .gmw-menu-item {
+    padding-left: "16px";
```

Comment:
Try to use less magic numbers. Use a multiple of `$iui-baseline`. Also drop the quotes.